components/core/Accordian.js

```diff
@@ -0,0 +1,34 @@
+const AbstractComponent = require('../AbstractComponent');
+
+/**
+ * The Accordian Core Component. In addition to the Component methods, this component provides several 
+ * convience methods to set known properties. 
+ * 
+ * Based on the documentation found at:
+ * https://github.com/adobe/aem-core-wcm-components/tree/master/content/src/content/jcr_root/apps/core/wcm/components/accordion/v1/accordion
+ */
+class Accordian extends AbstractComponent {
+
+    constructor() {
+        super();
+        this.props["sling:resourceType"] = "core-components-examples/components/accordion/v1/accordian";
```

Comment:
This isn't _quite_ the correct resource type. This will be a bit of a challenge question: you can check out the title component to see, but the challenge is to understand why it's a different resource type.